quiz/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+
+import { useReduce } from "./Reducer-context";
+
+import { Question } from "./Question";
+import { Routes, Route } from "react-router-dom";
+import { Login } from "./Login";
+import { Home } from "./Home";
+import { Privateroute } from "./Privateroute";
+import { Learninggraph } from "./Learninggraph";
+import { Navbar } from "./Navbar";
+import { User } from "./User";
+import {Signup} from "./Signup";
+export default function App() {
+  console.log("render");
+  console.log(useReduce());
```

Comment:
Just remove the console logs after the development is done. There are many such files where I have seen you use console logs so just remove them from all the files.